joomla/joomla-cms
# [4.0] Solving Duplicate ID ( #25088 ) Multilingual

Pull Request for Issue #25088 .

### Summary of Changes
In https://github.com/joomla/joomla-cms/pull/22166 we solved com_associations side by side "Bug1" by adding `id` to `$hiddenFields` in `layouts/joomla/edit/global.php`.
This only concerned articles when `Show Publishing Options` was set to `Hide` in Articles Options.

As found out in #25088, this creates a duplicate ID.
Example: see source in `administrator/index.php?option=com_tags&view=tag&layout=edit`
To solve the issue, this PR adds the `$hidden_fields` directly in the article edit tmpl and takes it off from global.php.


### Testing Instructions
Patch a clean install. (This can be done using Patchtester on a nighlty build)
Create a multilingual site by installing 2 languages and using the Multilingual Sample Data module in CPanel.
Set `Show Publishing Options` to `Hide` in Articles Options.
Load com_associations and set the filters to Articles and one of the content languages.
Click on the title of one of the articles to make sure the side by side will not loop.
Load `administrator/index.php?option=com_tags&view=tag&layout=edit` and verify that we have only one `jform_id`

### After patch
All still works fine for side by side in com_associations.
No more duplicate ID
